src/renderer/components/NewTeamModal.tsx

```diff
@@ -6,15 +6,15 @@ import React from 'react';
 import {Modal, Button, FormGroup, FormControl, FormLabel, FormText} from 'react-bootstrap';
 import {FormattedMessage, injectIntl, IntlShape} from 'react-intl';
 
-import {TeamWithIndex} from 'types/config';
+import {MattermostTeam} from 'types/config';
```

Comment:
Are this modal and these types eventually going to be renamed to "server" to match these other changes?

Reply:
Yes, there is a ticket to do this: https://github.com/mattermost/desktop/issues/623